### DISCUSSION TITLE
Unable to filter PRs authored by Dependabot

### DISCUSSION THREAD

**Discussion by adamdmharvey** [community/community#13716] [Mobile]
In the Web UI, you can filter pull requests authored by Dependabot. There must be a little magic behind it as technically the user id is `app/dependabot`.

<img width="301" alt="image" src="https://user-images.githubusercontent.com/33203301/160465618-b027c836-0523-4ad8-a5cb-67e16e52f732.png">

However, it is not possible to filter by this user in the current version of the app on iOS (v1.57.0). Starting to type "dep" searches only for actual accounts with that name. It would be nice to see Dependabot authored PRs using this author method in the app. (though I could also search for chore in subject, etc.)

| Mobile Screen Shot | Mobile Screen Shot 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/33203301/160466042-b35d22a3-93b1-4bb7-b4fd-d8ed314e4952.png) | ![image](https://user-images.githubusercontent.com/33203301/160466057-9c452f37-12b9-4bb2-b246-d6ec00f4744e.png) |

**Comment by brianlovin** [comment]
Wow, great spot on this discrepancy! I've flagged this to our team and we'll take a look 😊
